iothub/device/src/Microsoft.Azure.Devices.Client.csproj

```diff
@@ -79,6 +79,7 @@
     <Compile Include="ModernDotNet\HsmAuthentication\Transport\HttpRequestResponseSerializer.cs" />
     <Compile Include="ModernDotNet\HsmAuthentication\Transport\UnixDomainSocketEndPoint.cs" />
     <PackageReference Include="DotNetty.Codecs.Mqtt" Version="0.7.0" />
+    <PackageReference Include="DotNetty.Common" Version="0.7.1" />
```

Comment:
Do we want to consider adding a comment that this is temporarily needed until DotNetty.Codecs.Mqtt updates its reference?